### ISSUE TITLE
WebRTC Blocker not working properly

### ISSUE THREAD

**Issue by SilkArt** [uBlockOrigin/uAssets#8906]
### URL(s) where the issue occurs

https://webbrowsertools.com/test-webrtc-leak

### Describe the issue

When I enable the "Prevent WebRTC from leaking local IP addresses, then go to [](https://webbrowsertools.com/test-webrtc-leak/) to test the option,
I get the following message: Your browser is leaking webrtc ip-address!

### Screenshot(s)

![webrtc](https://user-images.githubusercontent.com/62780210/115097571-deee8000-9edf-11eb-89c2-cb88a3fdd3bd.png)

### Versions

- Browser/version: Chrome 89.0.4389.95
- uBlock Origin version: 1.34.0

### Settings

- Enabled all options under privacy

### Notes
I have run a WebRTC leak test multiple times and it still says, "Your local IP Address is being leaked."

**Comment by uBlock-user** [comment]
> Your browser is leaking webrtc ip-address!

Yes, your public IP address is not blocked only your local IP address, you're mistaking public IP for local IP. Working as expected.
